### ISSUE TITLE
Passing prop `className` to `FormTab` component has no effect

### ISSUE THREAD

**Issue by johncalvinroberts** [marmelab/react-admin#2936]
**What you were expecting:**
`className` prop to be passed to underlying material ui component

**What happened instead:**
the prop had no effect

**Steps to reproduce:**

**Related code:**
```
          <FormTab label="Detail" className={classes.tabInner}>
             ...
          </FormTab>
```

There is no way to style the `<TabbedForm/>` nor `<FormTab/>` components

```Warning: Material-UI: the key `tabInner` provided to the classes property is not implemented in TabbedForm.
You can only override one of the following: errorTabButton```

**Comment by fzaninotto** [comment]
The `className` passed to `FormTab` is passed to material-ui `<Tab>`, and serves to style the tab *header*. 

We'll accept an enhancement PR adding a new `contentClassName` prop to `FormTab` and `Tab`.

**Comment by fzaninotto** [comment]
Fixed by #2996
